tests/utils/stl/test/file_parsing.py

```diff
@@ -138,11 +138,15 @@ def parse_result_file(filename: Union[str, bytes, os.PathLike]) \
     res = dict()
     for line in parse_commented_file(filename):
         m = _EXPECTED_RESULT_REGEX.match(line)
+        if m is None:
+            raise Exception(f"Line \"{line}\" has incorrect format in {filename}.")

```

Comment:
These are good error messages but because the line can be long, I think we should front-load the description of what is bad about it.